std_msgs/Header header
int64 fake_altimeter     # fake_altimeter reading

